test/integration/test_events.py

```diff
@@ -10,6 +10,16 @@
 from ansible_runner import run, run_async
 
 
+@pytest.fixture(scope='session')
+def skipif_pre_ansible29():
```

Comment:
based on the fixture's definition, looks like this method should actually be name `skipif_pre_ansible28`. Am I reading this correctly?

```python
if LooseVersion(pkg_resources.get_distribution('ansible').version) < LooseVersion('2.8'):
    pytest.skip(reason="Valid only on Ansible 2.8+")
```

Reply:
apart from that question, looks like a great / helpful fixture 💯

Reply:
oops you are right